src/set_theory/cardinal_ordinal.lean

```diff
@@ -641,6 +641,16 @@ calc  #(list α)
 theorem mk_list_eq_omega (α : Type u) [encodable α] [nonempty α] : #(list α) = ω :=
 mk_le_omega.antisymm (omega_le_mk _)
 
+theorem mk_list_eq_mk_add_omega (α : Type u) [nonempty α] : #(list α) = #α + ω :=
```

Comment:
I think `max` is nicer to work with than `+`?

Reply:
I've used add for all things Loewenheim-Skolem related because that's the form I usually see that theorem stated in, but I've about come to the same conclusion. I might submit a separate PR reverting the other LS-related uses of `_ + omega` to `max _ omega`.